src/components/Dashboard/index.js

```diff
@@ -243,7 +243,7 @@ const Dashboard = ({ background = "white", textColor = "black" }) => {
               xs={12}
               sm={12}
               md={showOrgForm ? 6 : 12}
-              className="col-pad-24 pr-12 pt-8 pb-24 div-transition"
+              className="col-pad-24 pr-12 pt-8 pb-24 pl-12 div-transition"
```

Comment:
I think class 'col-pad-24' can be removed as it's property is being overridden.

Reply:
Thank you for pointing out. I will remove it.